### DISCUSSION TITLE
Is revanced.io legit? Are apk file secure to install?

### DISCUSSION THREAD

**Discussion by Megaman99** [ReVanced/revanced-discussions#899] [Question] [answered]
Hi everyone
I discovered revanced and i want to install it on my phone but i can't find an apk file here on github
I found the website but i don't know if it's legit and i don't know if those apk in the site are secure
Where can i download an apk of revanced and the microg app?

**Comment by oSumAtrIX** [comment, marked as answer]
There's some posts which answered this.

**Reply by Megaman99** [reply]
Which posts?
Can you link them please?

**Reply by Hasdan** [reply]
I believe they refer to discussion " Is there a Youtube revanced app ? #854 ", which asks the same question.
They specify that none of them owns this site, therefore it could be malicious.

Source: https://github.com/revanced/revanced-discussions/discussions/854

**Comment by kazimmt** [comment]
Fake Site

**Reply by Waleed-titi** [reply]
It is working and much easier like YouTube vanced you just download micro g and then download revanced application and then you have the application ready and it is really working and you don't have to build it and no complications.

**Reply by kazimmt** [reply]
> It is working and much easier like YouTube vanced you just download micro g and then download revanced application and then you have the application ready and it is really working and you don't have to build it and no complications.

It's working doesn't mean It's safe at all!
Patching app using ReVanced Manager is recommended
